[2024-11-15T06:35:41.189+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-15T06:35:41.203+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: football_data.extract_data manual__2024-11-15T06:35:37.326086+00:00 [queued]>
[2024-11-15T06:35:41.215+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: football_data.extract_data manual__2024-11-15T06:35:37.326086+00:00 [queued]>
[2024-11-15T06:35:41.215+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-15T06:35:41.240+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2024-11-15 06:35:37.326086+00:00
[2024-11-15T06:35:41.245+0000] {standard_task_runner.py:72} INFO - Started process 17779 to run task
[2024-11-15T06:35:41.253+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'football_data', 'extract_data', 'manual__2024-11-15T06:35:37.326086+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/football_dag.py', '--cfg-path', '/tmp/tmpcprxee9b']
[2024-11-15T06:35:41.256+0000] {standard_task_runner.py:105} INFO - Job 46: Subtask extract_data
[2024-11-15T06:35:41.305+0000] {task_command.py:467} INFO - Running <TaskInstance: football_data.extract_data manual__2024-11-15T06:35:37.326086+00:00 [running]> on host ip-172-31-28-221.eu-north-1.compute.internal
[2024-11-15T06:35:41.400+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aman Singh' AIRFLOW_CTX_DAG_ID='football_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-15T06:35:37.326086+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-15T06:35:37.326086+00:00'
[2024-11-15T06:35:41.401+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-15T06:35:41.577+0000] {logging_mixin.py:190} INFO - <Response [200]>
[2024-11-15T06:35:42.339+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow/pipelines/extraction_data.py", line 41, in main_extract_data
    'Stadium':clean_text(tds[0].text),
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow/pipelines/extraction_data.py", line 23, in clean_text
    text = text.replace(' â™¦',"")
           ^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'replace'
[2024-11-15T06:35:42.356+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=football_data, task_id=extract_data, run_id=manual__2024-11-15T06:35:37.326086+00:00, execution_date=20241115T063537, start_date=20241115T063541, end_date=20241115T063542
[2024-11-15T06:35:42.370+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-15T06:35:42.370+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 46 for task extract_data ('builtin_function_or_method' object has no attribute 'replace'; 17779)
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow/pipelines/extraction_data.py", line 41, in main_extract_data
    'Stadium':clean_text(tds[0].text),
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/airflow/pipelines/extraction_data.py", line 23, in clean_text
    text = text.replace(' â™¦',"")
           ^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'replace'
[2024-11-15T06:35:42.392+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-15T06:35:42.405+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-15T06:35:42.406+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
